inz-demo/internetnz/Issues/nethuiTicket/description.xml

```diff
@@ -0,0 +1,123 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo INZ Nethui Ticket</en>
+		<nl>Demo INZ Nethui Ticket</nl>
+	</Name>
+	<ShortName>
+		<en>Demo INZ Nethui Ticket</en>
+		<nl></nl>
+	</ShortName>
+	<SchemeManager>inz-demo</SchemeManager>
+	<IssuerID>internetnz</IssuerID>
+	<CredentialID>nethuiTicket</CredentialID>
+	<Description>
+		<en>INZ Nethui Ticket</en>
+		<nl></nl>
+	</Description>
+	<IssueURL>
+		<en></en>
+		<nl></nl>
+	</IssueURL>
+	<Attributes>
+		<Attribute id="id">
+			<Name>
+				<en>Ticket number</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Ticket number</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="event">
+			<Name>
+				<en>Event</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Event</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="eventDate">
+			<Name>
+				<en>Event date</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Event date</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="ticketType">
+			<Name>
+				<en>Ticket type</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Ticket type</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="fullName">
+			<Name>
+				<en>Full name</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Full name</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="email">
+			<Name>
+				<en>E-mail</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>E-mail</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="organisationName" required="false">
+			<Name>
+				<en>Organisation name</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Organisation name</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="membershipNumber" required="false">
+			<Name>
+				<en>INZ membership number</en>
```

Comment:
We already used our membership attribute on the phone already to get the ticket, do we need to store it again?